src/clipboard-action-cut.js

```diff
@@ -0,0 +1,15 @@
+import select from 'select';
+import command from './common/command';
+
+/**
+ * Cut action wrapper.
+ * @param {HTMLElement} target
+ * @return {String}
+ */
+const ClipboardActionCut = (target) => {
+  const selectedText = select(target);
+  command('cut');
```

Comment:
I don't know if it's a Promise, but in case it is considered asynchronous `command('cut')` to ensure it will be exec. before return the `selectedText`.

Reply:
Actually, it is a sync command at the browser level. It's based on the `document.execCommand()` approach. It requires first to select a text in the document, and then the command intention is executed. I guess that a smarter way would be using Async Clipboard, which is planned to be adopted as soon as we isolate the main actions. Any thoughts?